### ISSUE TITLE
wrong argument passed to perform_grid_search

### ISSUE THREAD

**Issue by mjvakili** [hudson-and-thames/research#116]
In [Chapter4/Trend-Follow-Question-Extended-SB-Cross-Val.ipynb](https://github.com/hudson-and-thames/research/blob/master/Chapter4/Trend-Follow-Question-Extended-SB-Cross-Val.ipynb), there is a slight problem with the arguments passed to the `perform_grid_search` function. 
The scoring argument needs to be set to a [scikit-learn metric](https://scikit-learn.org/stable/modules/classes.html#sklearn-metrics-metrics) such as `f1_score` [see the link](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html#sklearn.metrics.f1_score) and not a string such as `"f1"`.

**Comment by Jackal08** [comment]
Hi @mjvakili, 

Thanks, that code runs on an older version of the package. We have since updated it. We have made a ticket to update the documentation but won't be going back to fix previous tutorials. 

Best regards
Jacques

**Comment by mjvakili** [comment]
Hi @Jackal08, Thanks! I see. That makes sense as I had to modify quite a few lines in order to make some of the tutorials compatible with the package. Best, Mohammadjavad

**Comment by robolth** [comment]
Hi @mjvakili, may I know if you had to modify something else apart from the scoring argument to make it work? I'm trying to make it run myself and am having difficulties... thanks in advance!

**Comment by mjvakili** [comment]
Hi @robolth I had a quick look at it. You also need to set the scoring argument of `mean_decrease_impurity`, `mean_decrease_accuracy` and `single_feature_importance` to f1_score that comes from sklearn.metrics. 

But overall, if you just take a look at the implementations in the mlfinlab library it is quite clear what needs to be changed. The required modifications are usually very minor. I hope this helps...and sorry for the late reply.

**Comment by robolth** [comment]
Thanks!
